closed/src/java.base/share/classes/sun/security/ec/NativeXDHKeyPairGenerator.java

```diff
@@ -111,6 +111,7 @@ private void initializeImpl(XECParameters params, SecureRandom random) {
         ops = new XECOperations(params);
         this.random = (random != null) ? random : JCAUtil.getSecureRandom();
 
+        initializeJavaImplementation();
```

Comment:
We get here via the constructor, but before `lockedParams` is set leading to incorrect behavior in `initializeJavaImplementation()`.